src/ncp/ncp_spinel.cpp

```diff
@@ -473,12 +481,97 @@ otbrError NcpSpinel::HandleResponseForPropSet(spinel_tid_t      aTid,
     return error;
 }
 
+otbrError NcpSpinel::HandleResponseForPropInsert(spinel_tid_t      aTid,
+                                                 spinel_command_t  aCmd,
+                                                 spinel_prop_key_t aKey,
+                                                 const uint8_t    *aData,
+                                                 uint16_t          aLength)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    switch (mWaitingKeyTable[aTid])
+    {
+    case SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE:
+        if (aCmd == SPINEL_CMD_PROP_VALUE_IS)
+        {
+            spinel_status_t status = SPINEL_STATUS_OK;
+
+            VerifyOrExit(aKey == SPINEL_PROP_LAST_STATUS, error = OTBR_ERROR_INVALID_STATE);
+            SuccessOrExit(error = SpinelDataUnpack(aData, aLength, SPINEL_DATATYPE_UINT_PACKED_S, &status));
+            otbrLogWarning("Failed to subscribe to multicast address on NCP, error:%s", spinel_status_to_cstr(status));
```

Comment:
This warning is logged if the above code succeeds? Maybe you want to log the warning at exit if the error is not OTBR_ERROR_NONE?

Reply:
I see, thanks